cli/src/android/run.ts

```diff
@@ -19,7 +19,8 @@ export async function runAndroid(
     selectedTarget,
   );
 
-  const gradleArgs = ['assembleDebug'];
+  const arg = `assemble${config.android?.flavor || ''}Debug`;
```

Comment:
@sandor-biro You still need the word `Debug`, the correct argument is actually `assembleFlavorDebug` where `Flavor` is the value from the config

Reply:
Debug word is at the end of the template string, so if there is a flavor it will be assembleFlavorNameDebug and if not assembleDebug